tools/python/test_server/start_server.py

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env python3
 
+"""
+    Run from `$OMIM_ROOT/tools/python/test_server` dir
+"""
+
 import subprocess
 
 SERVER_RUNNABLE = 'server/testserver.py'
+LOG_FILE = 'server.log'
```

Comment:
It's better to use a random file name, and clean up it properly.

Reply:
What's the reason for random file name?

Reply:
To allow multiple instances to work together. Now they are launched in parallel and killed in 10 seconds timeout.

Reply:
Only one instance of the server will be launched for ctest.

Reply:
It is possible to launch several instances locally.

Reply:
And why'd you do this?
If you run them via start_server.py, nothing bad will happen.
However, if you run them via server/testserver.py... well, don't do this. 
Only 3-4 people from the entire OM community might want to run several instances locally, and they're aware of the consequences.
My point is not to overcomplicate things.